src/navigation/machines/oid4vciStateNavigation.tsx

```diff
@@ -135,6 +135,11 @@ const navigateAddContact = async (args: OID4VCIMachineNavigationArgs): Promise<v
       name: contact.contact.displayName,
       federations: federationParties,
       uri: contact.uri,
+      identities: contact.identities,
+      description: issuerBranding?.[0].description,
```

Comment:
Can't see the entire code, but can issuerBranding be an empty array?
```
issuerBranding?.[0]?.description
```

If so maybe something like
````
const branding = issuerBranding?.[0] || {}
navigation.navigate(MainRoutesEnum.OID4VCI, {
  screen: ScreenRoutesEnum.NEW_CONTACT_ADD,
  params: {
    name: contact.contact.displayName,
    federations: federationParties,
    uri: contact.uri,
    identities: contact.identities,
    description: branding.description,
    clientUri: branding.clientUri,
    tosUri: branding.tosUri,
    policyUri: branding.policyUri
````

Reply:
fair enough, changed